app/src/androidTest/java/org/simple/clinic/storage/migrations/Migration73AndroidTest.kt

```diff
@@ -0,0 +1,39 @@
+package org.simple.clinic.storage.migrations
+
+import org.junit.Test
+import org.simple.clinic.assertColumns
+
+class Migration73AndroidTest : BaseDatabaseMigrationTest(72, 73) {
+
+  @Test
+  fun `capabilities_should_be_added_to_the_user_table`() {
```

Comment:
Can we also add a test to make sure the `LoggedInUser ` data is retained after the migration?